workflows/cd.yml

```
@@ -1,7 +1,50 @@
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+
+      # Step 6: Authenticate with Google Cloud
+      - name: Authenticate with Google Cloud
+        run: |
+          gcloud auth activate-service-account --key-file=${{ secrets.GCP_SA_KEY }}
+          gcloud config set project ${{ secrets.GCP_PROJECT_ID }}
+
+      # Step 7: Deploy to Google App Engine
+      - name: Deploy to Google App Engine
+        run: |
+          gcloud app deploy --quiet
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)